eclipse-collections/src/main/java/org/eclipse/collections/impl/lazy/parallel/list/ListIterableParallelIterable.java

```diff
@@ -85,12 +85,6 @@ public T detect(Predicate<? super T> predicate)
     }
 
     @Override
-    public <V> ParallelListIterable<V> flatCollect(Function<? super T, ? extends Iterable<V>> function)
-    {
-        return new ParallelFlatCollectListIterable<T, V>(this, function);
-    }
-
```

Comment:
This method used to instantiate `ParallelFlatCollectListIterable` while its super class is instantiating `ParallelFlatCollectIterable `. Is this intentional to change the behaviour?